torchtext/prototype/t5/modules.py

```diff
@@ -582,3 +584,30 @@ def forward(
             return attn_output.transpose(1, 0), attn_output_weights, position_bias
         else:
             return attn_output, attn_output_weights, position_bias
+
+
```

Comment:
T5 model uses a root-mean-square layer normalization. `T5LayerNorm` implements this and is taken directly [from HF](https://github.com/huggingface/transformers/blob/360719a6a4e86c99cb852e844d30712eabb04baf/src/transformers/models/t5/modeling_t5.py#L239)